features/FEATURE_LWIP/lwip-interface/lwip-eth/arch/TARGET_NXP/TARGET_LPCTarget/lpc17_emac.c

```diff
@@ -611,7 +611,7 @@ static err_t lpc_low_level_output(struct netif *netif, struct pbuf *p)
 	/* Wait until enough descriptors are available for the transfer. */
 	/* THIS WILL BLOCK UNTIL THERE ARE ENOUGH DESCRIPTORS AVAILABLE */
 #if NO_SYS == 0
-	for (idx = 0; idx < dn; idx++) {
+	for (count = 0; count < dn; count++) {
```

Comment:
Can you just declare it in here, please.

    for (s_32_t count = 0; count < dn; count++) 

Minimise variable scopes where possible to make clear they don't have any wider significance.

Reply:
That is my preferred style, but I was following the style presented here. All local variables are declared at the top of the function.

That said, let me know if you still want me to make that change.

Reply:
I do like style consistency, but I like reduced-scope variables more. And as it's narrowly scoped, it has very little impact on global style...

Reply:
Done.